src/handbook/marketing/blog.md

```diff
@@ -25,7 +25,82 @@ Content creation and blogging are effective ways to communicate with our communi
 1. When both reviews are done*, the Author or SEO reviewer can merge. In the event of an external author, SEO/content review will merge.
 1. Once merged, SEO Reviewer will promote on FlowFuse Social channels. Check the [social media planning and posting section](/handbook/marketing/social-media/#planning-and-posting) for guidance and considerations.
 
-*It's ok to merge without a blog tile, as long as there's an art request open with a clear deadline and it doesn't get shared on socials without having the image.
+## Temporary Blog Tile Image Process
+
+When no designer is available or the content is time-sensitive, you may create a temporary blog tile so we can publish without delay.  
+Please still open an art request so a designer can update the tile later.
+
+Below are the approved scenarios.
+
+### 1. Use the Default Blog Tile (Recommended for Speed)
+
+If you don’t add an image to the article front matter, a default branded blog tile will be applied automatically.  
+This is always the fastest way to publish an article.
+
+There are two possible variations depending on whether a social post is needed:
+
+#### 1.1 Article Only (No Social Post Needed)
+
+Use this when urgency is driven mainly by SEO/indexing or internal timelines.
+
+- The default branded tile is enough for the initial publication.  
+- No additional assets are required at this stage.  
+
+#### 1.2 Article + Quick Social Post (Reuse Existing Assets)
+
+Use this when the article can publish with the default tile, but you need something quick for the social post.
+
+- If the article already contains screenshots, diagrams, or videos, you may reuse those assets for the social post.  
+- This keeps the blog tile simple (default) while allowing you to attach something relevant and fast for social sharing.
+
+
+### 2. Generate a Quick Tile With Nano Banana Pro
+
+Use this only when:
+
+- You need to publish a social post urgently **and**  
+- The article does not contain any usable assets.
```

Comment:
I would default to using this before we use up any other asset, if the result isn't needed, we fall back to human design. I think AI first provides 
1. Speed
2. Async, non-blocking, assets

For enhanced quality I think we should depend on a human, but only if we believe the quality is needed.